### PR TITLE
Newcomers_Guide.rst: Commands to Send Changes

### LINKED ISSUE #5403: Add git commit to Newcomers guide
Part of [Step 4 (Sending Your Changes)](http://api.coala.io/en/latest/Developers/Newcomers_Guide.html#step-4-sending-your-changes) is "create a commit"

But the command to do this is not shown.

### PR DESCRIPTION
**Pull request by rajdeepbharati**
Closes #5403 

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by rajdeepbharati** [comment]
Now I have mentioned the issue #5403 in the first commit message

**Comment by rajdeepbharati** [comment]
I have fixed the commit message. Could you please tell me if the message is in the proper format? I'm new to this organisation.

**Comment by li-boxuan** [comment]
Please read https://coala.io/newcomer 😉

**Comment by rajdeepbharati** [comment]
Changed.

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -392,6 +392,18 @@ Step 4. Sending Your Changes
    Your current branch will have an asterisk (\*) next to it. Ensure that there
    is no asterisk next to the master branch.
 
+Add your changes:
+
+::
+
+    $ git add .
```
Comment (li-boxuan) [inline docs/Developers/Newcomers_Guide.rst]: This is prone to error. Very likely, newcomers add some random files to the repository, and by using `git add .` they add random files and commit.

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -392,6 +392,18 @@ Step 4. Sending Your Changes
...
+
+Commit your changes using the following commands:
```
Comment (li-boxuan) [inline docs/Developers/Newcomers_Guide.rst]: You only provide one command below, but you use plural form here

File: docs/Developers/Newcomers_Guide.rst [hunk 3]
```diff
@@ -392,6 +392,18 @@ Step 4. Sending Your Changes
...
+
+::
+
+    $ git commit -m "your commit message"
+
 Now that you've fixed the issue, you've tested it, and you think it is ready
 to be merged, create a commit and push it to your fork, using:
```
Comment (li-boxuan) [inline docs/Developers/Newcomers_Guide.rst]: Your instructions should reside somewhere here, e.g. between `create a commit` and `push it to your fork`.

**Comment by rajdeepbharati** [comment]
Could you please check it again?

File: docs/Developers/Newcomers_Guide.rst [hunk 4]
```diff
@@ -393,7 +393,19 @@ Step 4. Sending Your Changes
...
+::
+
+    $ git add <file/folder_name>
+
+Create a commit using the following command:
+
+::
+
+    $ git commit
+
```
Comment (akshatkarani) [inline docs/Developers/Newcomers_Guide.rst]: here add that this will open a text editor to write commit message.

Comment (rajdeepbharati) [reply docs/Developers/Newcomers_Guide.rst]: Done

File: docs/Developers/Newcomers_Guide.rst [hunk 5]
```diff
@@ -393,7 +393,20 @@ Step 4. Sending Your Changes
...
+This will open a text editor to write commit message.
+After writing the message, push it to your fork, using:
```
[the remark this answers is not in the document]
Comment (rajdeepbharati) [reply docs/Developers/Newcomers_Guide.rst]: Changed

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**